modules/machine-health-checks-creating.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * master/deploying-machine-health-checks.adoc
+
+[id='machine-health-checks-creating-{context}']
+= Creating a `MachineHealthCheck` resource
+
+You can create a `MachineHealthCheck` resource for all `MachinePools` in your
+cluster except the `master` pool.
+
+.Prerequisites
+
+* Install the `oc` command line and kubectl.
+
+.Procedure
+
+. Create a *_healthcheck.yml_* file that contains the definition of your
```

Comment:
Is there a better method for applying a health check?

Reply:
There is not right now unless the resource is created by the installer itself.

Reply:
Do you expect that the resource will be created by the installer itself in the future?

Reply:
nope any time soon. It's a post install feature

Reply:
Great! Then if that ever happens, let me know so I can update the docs.